### ISSUE TITLE
eslint adding unnecessary indents

### ISSUE THREAD

**Issue by LJNeon** [eslint/eslint#9565]
**Tell us about your environment**

* **ESLint Version:** 4.10.0
* **Node Version:** 9.0.0
* **npm Version:** 5.5.1

**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
	"env": {
		"es6": true,
		"node": true
	},
	"extends": "eslint:recommended",
	"rules": {
		"indent": [2, "tab"],
		"linebreak-style": [2, "unix"],
		"quotes": [2, "double"],
		"semi": [2, "always"]
	},
	"parserOptions": {
		"sourceType": "module",
		"ecmaVersion": 2017
	}
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const colors = {
	red: 0xFF4136,
	orange: 0xFF851B,
	yellow: 0xFFDC00,
	lime: 0x01FF70,
	green: 0x2ECC40,
	olive: 0x3D9970,
	teal: 0x39CCCC,
	aqua: 0x7FDBFF,
	blue: 0x0074D9,
	navy: 0x001F3F,
	maroon: 0x85144B,
	fuchsia: 0xF012BE,
	purple: 0xB10DC9,
	black: 0x111111,
	grey: 0xAAAAAA,
	silver: 0xDDDDDD,
	white: 0xFFFFFF
}, groups = {
	good: [colors.lime, colors.green, colors.olive, colors.teal, colors.aqua, colors.blue, colors.navy],
	neutral: [colors.black, colors.grey, colors.silver, colors.white],
	bad: [colors.red, colors.orange, colors.yellow, colors.maroon, colors.fuchsia, colors.purple]
}, random = require("../utilities/random");
```

```bash
eslint src/**
```

**What did you expect to happen?**
It wouldn't error as the indents are fine.
**What actually happened? Please include the actual, raw output from ESLint.**
It complained about the indents and wanted 1 more indent added than was needed.

**Comment by gyandeeps** [comment]
Please share the exact eslint output.

**Comment by LJNeon** [comment]
```
   3:1  error  Expected indentation of 2 tabs but found 1  indent
   4:1  error  Expected indentation of 2 tabs but found 1  indent
   5:1  error  Expected indentation of 2 tabs but found 1  indent
   6:1  error  Expected indentation of 2 tabs but found 1  indent
   7:1  error  Expected indentation of 2 tabs but found 1  indent
   8:1  error  Expected indentation of 2 tabs but found 1  indent
   9:1  error  Expected indentation of 2 tabs but found 1  indent
  10:1  error  Expected indentation of 2 tabs but found 1  indent
  11:1  error  Expected indentation of 2 tabs but found 1  indent
  12:1  error  Expected indentation of 2 tabs but found 1  indent
  13:1  error  Expected indentation of 2 tabs but found 1  indent
  14:1  error  Expected indentation of 2 tabs but found 1  indent
  15:1  error  Expected indentation of 2 tabs but found 1  indent
  16:1  error  Expected indentation of 2 tabs but found 1  indent
  17:1  error  Expected indentation of 2 tabs but found 1  indent
  18:1  error  Expected indentation of 2 tabs but found 1  indent
  19:1  error  Expected indentation of 2 tabs but found 1  indent
  20:1  error  Expected indentation of 1 tab but found 0   indent
  21:1  error  Expected indentation of 2 tabs but found 1  indent
  22:1  error  Expected indentation of 2 tabs but found 1  indent
  23:1  error  Expected indentation of 2 tabs but found 1  indent
  24:1  error  Expected indentation of 1 tab but found 0   indent
```
here's the exact eslint output

**Comment by not-an-aardvark** [comment]
This is working as intended.

**Comment by LJNeon** [comment]
Could you explain why two indents is intended when only one should be?

**Comment by not-an-aardvark** [comment]
This is a subjective stylistic choice, so I don't think either version is objectively more correct than the other.

For reasons I'm unsure about, the original author of the `indent` rule chose to require indentation for variable declarations with multiple declarators, and the rule has been doing this for several years. During that time, many users have written code with indentation like that, so changing the decision would break many peoples' builds.

So the reason is basically that the rule has enforced this for awhile, and changing the behavior would have costs without any significant benefits.

**Comment by LJNeon** [comment]
that's pretty annoying and makes it so I can't use the rule at all, but I see your point.

**Comment by platinumazure** [comment]
@not-an-aardvark Could the ignoredNodes option be used here to not enforce variable declarations with multiple declarators, so that it would be possible to enforce the preferred indentation through a custom rule or plugin?

**Comment by not-an-aardvark** [comment]
Yes, I think using the `VariableDeclaration` selector for `ignoredNodes` might work.
